Lykke.Service.LykkeService/src/Lykke.Job.LykkeService/PeriodicalHandlers/MyPeriodicalHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Common;
+using Lykke.Common.Log;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Autofac;
+
+namespace Lykke.Job.LykkeService.PeriodicalHandlers
+{
+    public class MyPeriodicalHandler : IStartable, IStopable
+    {
+        private readonly TimerTrigger _timerTrigger;
+        
+        public MyPeriodicalHandler(ILogFactory logFactory)
+        {
+            // TODO: Sometimes, it is enough to hardcode the period right here, but sometimes it's better to move it to the settings.
+            // Choose the simplest and sufficient solution
+            _timerTrigger = new TimerTrigger(nameof(MyPeriodicalHandler), TimeSpan.FromSeconds(10), logFactory);
+            _timerTrigger.Triggered += Execute;
+        }
+
+        public void Start()
+        {
+            _timerTrigger.Start();
+        }
+        
+        public void Stop()
+        {
+            _timerTrigger?.Stop();
```

Comment:
?.Stop() could be replaced with .Stop(), since _timerTrigger is initialized in ctor